plutus-tx/src/Language/PlutusTx/Applicative.hs

```diff
@@ -0,0 +1,43 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+module Language.PlutusTx.Applicative where
+
+import           Language.PlutusTx.Functor
+import           Prelude                   (Maybe (..))
+
+infixl 4 <*>, <*, *>
+
+class Functor f => Applicative f where
+    {-# MINIMAL pure, (<*>) #-}
+    -- | Lift a value.
+    pure :: a -> f a
+
+    -- | Sequential application.
+    --
+    -- A few functors support an implementation of '<*>' that is more
+    -- efficient than the default one.
```

Comment:
I don't understand this comment.

Reply:
Uh, this is just me copying comments from `base`. I'll get rid of them.